Tests/FluentAssertions.Specs/Extensions/ObjectExtensionsSpecs.cs

```diff
@@ -60,8 +59,8 @@ public void When_comparing_a_numeric_to_a_numeric_it_should_succeed(object actua
     public static IEnumerable<object[]> GetNumericAndNumericData()
     {
         return from x in GetNumericIConvertibles()
-               from y in GetNumericIConvertibles()
-               select new[] { x, y };
+            from y in GetNumericIConvertibles()
+            select new[] { x, y };
```

Comment:
I like that too, but I couldn't find a setting that would trigger Rider/R# to format it as such. And aligning as you had in your original example looks ugly on my machine, since I prefer to use a proportional font.